### PR TITLE
Add commons. Make local dev possible. And fix pstat 4601

### PR DESCRIPTION
**Pull request by deti**
* Add commons. 
* Make local dev possible. 
* And fix pstat 4601

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -90,7 +89,6 @@ They are kept in separate repos and updated manually by coping:
 * removenbsp
 * `structuredheadings <https://github.com/PolicyStat/ckeditor-plugin-structured-headings>`_
 
-These plugins packed together with CKEditor by symlinking during  update_ckeditor.py run.
```
Comment (bocribbz) [inline README.rst:93]: does `update_ckeditor.py` need to be updated to not symlink them now?

Comment (bocribbz) [inline README.rst:93]: does `update_ckeditor.py` need to be updated to not symlink them now?

File: plugins/structuredheadings/plugin.js [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+
+					// Fix for PolicyState issue 4601
+					// Notes on what `value !== "clear"` fixes
+					// Somve this function called when we do `Clear Numbering` in PolicyStat fork
+					// reapplyStyle behaviour
+					// 1) Clear all styles
+					// 2) Reapply style passed in `value`
+					// However, there's no style with name `clear`. And we falling back to the default style
+					// We just don't need to reapplyStyle for `clear` because proper styles already there.
+					if (value !== "clear") {
```
Comment (deti) [inline plugins/structuredheadings/plugin.js:825]: @bocribbz custom code is only this if. It fixes 4601

File: README.rst (hunk 1 above)
Comment (deti) [reply README.rst:93]: Yup. `update_ckeditor.py` would only copy now. But I wanted to pass this PR first before pushing changes in PolicyStat

Comment (deti) [reply README.rst:93]: Yes. It's removed https://github.com/PolicyStat/PolicyStat/pull/4601/files#diff-6e6b478860ccd1b74329c46ffc3739a2L37

**Review by bocribbz — APPROVED** [review]
👍

**Pull request merged**